Neos.ContentRepository.Core/Classes/Feature/WorkspaceCommandHandler.php

```diff
@@ -358,14 +353,7 @@ private function handleRebaseWorkspace(
             $commandHandlingDependencies
         );
 
-        $rebaseableCommands = RebaseableCommands::extractFromEventStream(
-            $this->eventStore->load(
-                ContentStreamEventStreamName::fromContentStreamId($workspace->currentContentStreamId)
-                    ->getEventStreamName()
-            )
-        );
-
-        if ($rebaseableCommands->isEmpty()) {
+        if (!$workspace->hasPublishableChanges()) {
```

Comment:
Same here, IMO we should make this case explicit, i.e. throw an exception if there is nothing to rebase